extensions/amp-install-serviceworker/0.1/amp-install-serviceworker.js

```diff
@@ -92,27 +92,20 @@ export class AmpInstallServiceWorker extends AMP.BaseElement {
     }
   }
 
+  /** @override  */
+  getPriority() {
+    return 2;
```

Comment:
We should have constants for this. At least give it a comment.

Reply:
Also, I think 3 (even lower than ads) may be more appropriate.

Reply:
Replaced with 3. Created a new issue #13555 to replace them with constants later